Practicum/Pract. 08/Vector.cpp

```diff
@@ -0,0 +1,163 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include "Vector.h"
+#include <new>
+#include <exception>
+
+Vector::Vector() :Vector(4) {}

```

Comment:
What is 4? Should be constant which has name.
Also should be inside a namespace/static